tools/buildtools/topologyeditors/add_top.xml

```diff
@@ -0,0 +1,129 @@
+<tool id="modify_topology" name="Adding New Topology Information" version="@TOOL_VERSION@+galaxy@GALAXY_VERSION@">
+    <description>to a GROMACS topology file</description>
+
+    <macros>
+        <import>macros.xml</import>
+        <token name="@GALAXY_VERSION@">0</token>
+    </macros>
+
+    <expand macro="requirements">
+       <requirement type="package" version="3.8">python</requirement>
+    </expand>
+
+    <command detect_errors="exit_code"><![CDATA[
+        #if $str($functionality.what2add) == "mol":
+            python3 $__tool_directory__/newmolparameters.py --top_file '$functionality.inputtopology' --atom_file '$functionality.nonbondparam' --mol_file '$functionality.bondparam' --out '$output1'
```

Comment:
please single-quote all paths ... e.g. `'$__tool_directory__/newmolparameters.py'`

and below

Reply:
made change